lib/control/v1/index.js

```diff
@@ -13,6 +14,10 @@ exports.attach = function (app, storage) {
   app.get('/v1/cubbyhole/:token/:path', require('./cubbyhole')(storage));
   app.get('/v1/credential/:token/:mount/:role', require('./credential')(storage));
 
+  app.use(BodyParser.json());
```

Comment:
Global preprocessing middleware should be defined before any routes. This is a 3-day head scratcher waiting to happen for anyone who doesn't have a firm grasp on how Express processes requests.